packages/@o3r/core/ng-package.json

```diff
@@ -7,6 +7,6 @@
     "entryFile": "src/public_api.ts"
   },
   "allowedNonPeerDependencies": [
-    "chokidar", "comment-json", "globby", "semver", "tslib", "uuid", "@o3r/schematics"
+    "chokidar", "comment-json", "globby", "semver", "tslib", "uuid", "@o3r/schematics", "@angular-devkit/core", "@schematics/angular"
```

Comment:
Not sure why it is an embedded dependencies.
If I am not mistaking, they are used only in old ng update